src/Rendering/prePassRenderer.ts

```diff
@@ -526,24 +655,46 @@ export class PrePassRenderer {
     private _update() {
         this._disable();
         let enablePrePass = false;
+        this._scene.imageProcessingConfiguration.applyByPostProcess = false;
 
         for (let i = 0; i < this._scene.materials.length; i++) {
             if (this._scene.materials[i].setPrePassRenderer(this)) {
                 enablePrePass = true;
+
+                this._setRenderTargetState(this.defaultRT, true);

```

Comment:
This line is in the loop but does not depend on it. Shouldn't the call be done after the loop if `enablePrePass = true`?

Reply:
It sure can be, it's just a boolean flag switch